internal/cmd/diag.go

```diff
@@ -34,3 +35,22 @@ func ExecInUvm(ctx context.Context, vm *uvm.UtilityVM, req *shimdiag.ExecProcess
 	err = cmd.Run()
 	return cmd.ExitState.ExitCode(), err
 }
+
+func ExecInHost(ctx context.Context, req *shimdiag.ExecProcessRequest) (int, error) {
```

Comment:
ExecOnHost sounds better to me but don't have strong feelings

Reply:
I changed it to ExecInShimHost, does that seem fair?

Reply:
More than fair, I like that more honestly 😃